test_scripts/API/Restructuring_OnResetTimeout/008_BC_OnResetTimeout_with_invalid_data.lua

```diff
@@ -0,0 +1,66 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
```

Comment:
`2) Some time after receiving RPC request on HMI is passed` looks very vague/unclear to me since "some time` is unknown

I like the description in your previous test scripts
 ` 3) HMI sends BC.OnResetTimeout(resetPeriod = 6000) with wrong requestID to SDL in 6 sec after HMI request`

so I suggest to remove  `2) Some time after receiving RPC request on HMI is passed` 
and update `3) HMI sends invalid BC.OnResetTimeout(resetPeriod = 13000)` 
to something like 
`3) HMI sends invalid BC.OnResetTimeout(resetPeriod = 13000) in 6 sec after HMI received the request`

Reply:
this comment apply to all test scripts that have "some  time after ..."